apps/studio/components/interfaces/Settings/Addons/PITRSidePanel.tsx

```diff
@@ -71,7 +71,7 @@ const PITRSidePanel = () => {
   const { data: databases } = useReadReplicasQuery({ projectRef })
   const { data: addons, isLoading } = useProjectAddonsQuery({ projectRef })
   const { data: subscription } = useOrgSubscriptionQuery({ orgSlug: organization?.slug })
-  const hasHipaaAddon = subscriptionHasHipaaAddon(subscription)
+  const hasHipaaAddon = true
```

Comment:
This change needs to be reverted

Reply:
We should also check for project settings `is_sensitive` flag - if that's false, we can skip all HIPAA check and treat it as regular project